packages/lib/services/rest/routes/notes.test.ts

```diff
@@ -78,4 +78,44 @@ describe('routes/notes', () => {
 
 		expect(response).toBe('');
 	});
+
+	test('should not copy content from invalid protocls', async () => {
+		const url = 'file:///home/user/file.db';
+		const shimFsDriverCopySpy = jest.fn();
+		jest.spyOn(shim, 'fsDriver').mockImplementation(() => {
+			return {
+				copy: shimFsDriverCopySpy,
+			} as any;
+		});
```

Comment:
I would prefer if we avoid using `spy` as much as possible because it tests implementation details, and the more we do this the harder it becomes to refactor the code. For example if we start using `fs-extra` instead of `fsDriver` this test would be broken but there's no reason for that to happen since the input/output of the function hasn't changed.

In that particular case, isn't it possible to verify that the function simply did nothing (didn't download any media file)?